Client/Ecosia/Experiments/Unleash/BingDistributionExperiment.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Core
+
+struct BingDistributionExperiment {
+    
+    private init() {}
+    
+    static func incrementCounter() {
+        BingDistributionExperimentCounter.increment()
+    }
+    
+    static func getCounterCurrentCount() -> Int {
+        BingDistributionExperimentCounter.read()
+    }
+    
+    static var isEnabled: Bool {
+        Unleash.isEnabled(.bingDistribution)
+    }
+    
+    static func makeBingSearchURLFromURL(_ sourceURL: URL) -> URL? {
+        var urlComponents = URLComponents(url: sourceURL.absoluteURL, resolvingAgainstBaseURL: false)
+        urlComponents?.host = "bing.com"
+        urlComponents?.queryItems = [
```

Comment:
By doing this we are replacing existing `queryItems` like `q=` which is what add the search input.